bindings/cs/rl.net/DecisionResponse.cs

```diff
@@ -0,0 +1,190 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using System.Runtime.InteropServices;
+
+using Rl.Net.Native;
+using System.Collections;
+
+namespace Rl.Net {
+    namespace Native
+    {
+        internal partial class NativeMethods
+        {
+            [DllImport("rl.net.native.dll")]
+            public static extern IntPtr CreateSlotResponse();
+
+            [DllImport("rl.net.native.dll")]
+            public static extern void DeleteSlotResponse(IntPtr slotResponse);
```

Comment:
This is not used, since the NativeObject never owns the handle; we should consider removing it.